eclair-core/src/main/scala/fr/acinq/eclair/db/Databases.scala

```diff
@@ -37,11 +45,15 @@ trait Databases {
 
   val pendingRelay: PendingRelayDb
 
-  def backup(file: File): Unit
+  def obtainExclusiveLock(): Unit
```

Comment:
The choice to make `obtainExclusiveLock` a method that can be implemented as a no-op, whereas `backup` is a separate trait is not very consistent. Also, changing `CanBackup`->`FileBackup` makes it clear that this is only one type of backup, there may be others. How about:

```scala
trait Databases {
  val network: NetworkDb
  val audit: AuditDb
  val channels: ChannelsDb
  val peers: PeersDb
  val payments: PaymentsDb
  val pendingRelay: PendingRelayDb
}

trait FileBackup { this: Databases =>
  def backup(file: File): Unit
}

case class SqliteDatabases(...) extends Databases with FileBackup {
  def backup(file: File): Unit = ???
}

case class PostgresDatabases(...) extends Databases {
  def obtainExclusiveLock(): Unit = ???
}
```